packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1332,16 +1331,17 @@ curl -v -X GET \
 ]
 ```
 
-
-
 #### List Users with a filter
 
 Lists all users that match the filter criteria. To ensure optimal performance, Okta recommends using a [search parameter](#list-users-with-search) instead of a filter.
 
 This operation:
 
-* Requires [URL encoding](http://en.wikipedia.org/wiki/Percent-encoding). For example, `filter=lastUpdated gt "2013-06-01T00:00:00.000Z"` is encoded as `filter=lastUpdated%20gt%20%222013-06-01T00:00:00.000Z%22`.
-* Supports only a limited number of properties: `status`, `lastUpdated`, `id`, `profile.login`, `profile.email`, `profile.firstName`, and `profile.lastName`.
+- Requires [URL encoding](http://en.wikipedia.org/wiki/Percent-encoding). For example, `filter=lastUpdated gt "2013-06-01T00:00:00.000Z"` is encoded as `filter=lastUpdated%20gt%20%222013-06-01T00:00:00.000Z%22`.
+- Supports the following limited number of properties: `status`, `lastUpdated`, `id`, `profile.login`, `profile.email`, `profile.firstName`, and `profile.lastName`.
+- Supports only the equal `eq` operator from the standard Okta API filtering semantics, except in the case of the `lastUpdated` property which can use the inequalilty operators (`gt`, `ge`, `lt`, and `le`).
+- Supports only the logical operators `and` and `or`.
+- Is case sensitive.
```

Comment:
Hm, well, parts of it are.  For example, I'm fairly sure the sort order `asc`/`desc` can be `ASC` or even `dEsC`.  Probably no need to go into that level of detail, but I want us to keep it in mind in case we see other parts where the difference in case sensitivity is more significant.

Reply:
I think my goal was to mention the values you're searching on are case-sensitive; that is, filtering on "John" or "john" will return different results. But it's open to interpretation the way I presented it, I agree. The core Okta API content on filtering goes into a bit more detail than I did. I've combined that content with what I wanted to present:

`Is case sensitive for attribute names and query values, while attribute operators are case-insensitive.`